src/components/vacation-requests-table/vacation-requests-table-toolbar/vacation-requests-table-toolbar.tsx

```diff
@@ -60,6 +64,13 @@ const TableToolbar = ({
   const language = useAtomValue(languageAtom);
   const adminMode = UserRoleUtils.adminMode();
   const { pathname } = useLocation();
+  const isToolbarVisible = toolbarOpen && !formOpen && selectedRowIds?.length;
+  const buttonLabel = isUpcoming ? strings.tableToolbar.future : strings.tableToolbar.past;
+  const singleSelectionSize = adminMode ? 3 : 6
+  const multiSelectionSize = adminMode ? 6 : 12
```

Comment:
I don't even know what these numbers are 3/6 & 6/12, are we even using them? to my knowledge nothing changes even on mobile view

Reply:
Feel free to remove if certain it is not needed, otherwise this is fine.